automake/post/rules/makedirs.am

```diff
@@ -0,0 +1,65 @@
+#
+#    Copyright 2020 Nest Labs Inc. All Rights Reserved.
```

Comment:
Minor nit: This should probably be `Copyright (c) 2020 Project nlbuild-autotools Authors`.